workspaces/rbac/.changeset/empty-spoons-prove.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage-community/plugin-rbac-backend': patch
+'@backstage-community/plugin-rbac-node': patch
```

Comment:
Why are these in the changeset? I dont see any cahnges related to these packages.